spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Users controller', type: :request do
+  describe 'GET /index' do
+    before :each do
+      get '/users'
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'the response includes right text' do
+      expect(response.body).to include('The list of users')
+    end
+
+    it 'the correct response status' do
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe 'GET /show' do
+    before :each do
+      get '/users/1'
+    end
+    it 'renders the correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'the response includes right text' do
+      expect(response.body).to include('The list of users\'s id')
+    end
+
+    it 'the correct status' do
+      expect(response.status).to eq(200)
+    end
+  end
+end
```

Comment:
- [ ]  I am getting two. Please fix the error.


![image](https://user-images.githubusercontent.com/38635158/161163013-ce8a12b1-06bb-4d1a-b512-0050cf2abef6.png)